python/plugins/processing/gui/Postprocessing.py

```diff
@@ -78,15 +78,15 @@ def handleAlgorithmResults(alg, context, feedback=None, showResults=True):
                     layer.loadNamedStyle(style)
                 details.project.addMapLayer(context.temporaryLayerStore().takeMapLayer(layer))
         except Exception:
-            QgsMessageLog.logMessage("Error loading result layer:\n" + traceback.format_exc(), 'Processing', QgsMessageLog.CRITICAL)
+            QgsMessageLog.logMessage(QCoreApplication.translate('PostProcessing', "Error loading result layer:\n") + traceback.format_exc(), 'Processing', QgsMessageLog.CRITICAL)
             wrongLayers.append(str(l))
         i += 1
 
     QApplication.restoreOverrideCursor()
     if wrongLayers:
-        msg = "The following layers were not correctly generated.<ul>"
+        msg = QCoreApplication.translate('PostProcessing', "The following layers were not correctly generated.<ul>")
```

Comment:
I would try to keep HTML codes or control sequences like `\n` out of translatable strings wherever possible